Add hint to activate the missing extension. (#51)
TYPO3-Documentation/TYPO3CMS-Exceptions: Documentation/Exceptions/1253191023.rst

.. include:: /If-you-encounter-this-exception.rst.txt

TypoScript object path "tt_content.list.20.your_extension" does not exist
=========================================================================

> Case 1 - Missing entry in Fluid Styled Content
> ----------------------------------------------

Reviewer: ```suggestion
Missing content rendering definition
------------------------------------
```

           templateName = YourListTemplateName
       }
   }

::

   TYPO3Fluid\Fluid\Core\ViewHelper\Exception
   TypoScript object path "tt_content.list.20.5" does not exist

> This means that you have inserted a frontend plugin, but the corresponding extension (tt_products) is not activated.

Reviewer: How is this possible? Assuming Legacy Mode (since all packages are automatically activated in Composer Mode) one cannot add a frontend plugin of an extension which is not activated.

Reviewer: Ah OK, now I get it. This could even happen in Composer Mode.

   }

> Case 2 - Not activated extension
> ---------------------------------

Reviewer: ```suggestion
Removed/inactive extension
--------------------------
```

::

   TYPO3Fluid\Fluid\Core\ViewHelper\Exception
   TypoScript object path "tt_content.list.your_extension." does not exist

> This means that you have formerly activated an extension. Then you have inserted a frontend plugin. 
> But at a later time you have deactivated this extension. You must either hide this plugin or activate the extension again.

Reviewer: ```suggestion
This could be cause by a frontend plugin from an extension which has been removed/deactivated in the meantime.

A possible fix is hiding/removing the plugin or installing/activating the related extension again.
```
